src/elementary/FundamentalParticleID.hpp

```diff
@@ -40,10 +40,15 @@ namespace elementary {
 
     /* methods */
 
+    /**
+     *  @brief return the fundamental particle za value
+     */
+    int za() const noexcept { return this->hash_; }
+
     /**
      *  @brief return the fundamental particle hash value
      */
-    int hash() const noexcept { return this->hash_; }
+    int hash() const noexcept { return this->za(); }

```

Comment:
Why have two names for this? Why not just have `za` and get rid of `hash`?

Reply:
I wanted the same interface for all identifiers. For the fundamental particle, the hash is the za. For the nucleus and the nuclide it is za * 100 + level. The particle identifier also makes a hash which is again different, depending on the underlying type.

Reply:
Maybe at some point we'll change the hash, and by already providing the function to distinguish between za and hash, I can avoid changing too much later.